source/MRMesh/MROffset.cpp

```diff
@@ -118,8 +118,12 @@ tl::expected<Mesh, std::string> offsetPolyline( const Polyline3& polyline, float
 
     std::vector<EdgeId> newHoles;
     newHoles.reserve( contours.size() );
-    for ( const auto& cont : contours )
+    for ( auto& cont : contours )
+    {
+        if ( cont.size() == 2 )
+            cont.push_back( cont.back() );
```

Comment:
please add a code comment why we duplicate back() point for 2-point contours. If it is a workaround for the next function, then probably better to fix it instread?